integration-tests/src/test/java/oracle/weblogic/kubernetes/ItIntrospectVersion.java

```diff
@@ -944,6 +1031,26 @@ private static void verifyMemberHealth(String adminServerPodName, List<String> m
         });
   }
 
+  private Map getPodsWithTimeStamps(String domainNamespace, String adminServerPodName,
```

Comment:
Can it be moved to utility class. This is being used in many tests when we verify a domain restart.

Reply:
Done